lib/system/environ/validate.go

```diff
@@ -45,9 +47,13 @@ func ValidateInstall(env *localenv.LocalEnvironment) func() error {
 			// Operational state directory requirements are advisory
 			log.WithError(err).Warn("Failed to validate state directory requirements.")
 		}
-		// FIXME: validate enough of directory contents to make an educated decision automatically
-		if err := validateDirectoryEmpty(stateDir); err != nil {
-			return trace.Wrap(err)
+		if err := validateNoActiveService(stateDir); err != nil {
```

Comment:
Is it safe to validate only the service is running, and not the contents of the stateDir?